src/components/Stats.vue

```diff
@@ -20,7 +20,7 @@ export default {
       return this.searchStore.query;
     },
     totalResults() {
-      return this.searchStore.totalResults;
+      return this.searchStore.totalResults.toLocaleString();
```

Comment:
I think we can have an argument to this function and forwarding the locale into the `toLocaleString()` call 🤔